server/game/cards/01_SOR/units/K2SOCassiansCounterpart.ts

```diff
@@ -0,0 +1,34 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { TargetMode } from '../../../core/Constants';
+
+export default class K2SOCassiansCounterpart extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '3232845719',
+            internalName: 'k2so#cassians-counterpart',
+        };
+    }
+
+    public override setupCardAbilities() {
+        // TODO: This doesn't support Twin Sun correctly
```

Comment:
Fine to leave this here, but for the future - there are so many places where this is true that we definitely shouldn't try to mark all of them